src/buildercore/bootstrap.py

```diff
@@ -460,23 +457,23 @@ def update_ec2_stack(stackname, context, concurrency=None, formula_revisions=Non
     script that can be downloaded from the web and then very conveniently
     installs it's own dependencies. Once Salt is installed we give it an ID
     (the given `stackname`), the address of the master server """
+
+    # no longer a snapshot of the project data at creation time, now it's
+    # mostly just top-level project attributes as we need them.
     pdata = context['project']
```

Comment:
should be renamed from `pdata` to something like `project_repositories` to avoid confusion with the "snapshot of the project data"